docs/06-concepts/18-testing/02-the-basics.md

```diff
@@ -127,11 +127,21 @@ The following optional configuration options are available to pass as a second a
 
 |Property|Type|Default|
 |:-----|:-----|:---:|
+|`applyMigrations`|`bool?`|`true`|
+|`enableSessionLogging`|`bool?`|`false`|
 |`rollbackDatabase`|`RollbackDatabase?`|`RollbackDatabase.afterEach`|
 |`runMode`|`String?`|`ServerpodRunmode.test`|
-|`enableSessionLogging`|`bool?`|`false`|
-|`applyMigrations`|`bool?`|`true`|
-|`testGroupTagsOverride`|`List<String>?`|`null`|
+|`serverpodLoggingMode`|`ServerpodLoggingMode?`|`ServerpodLoggingMode.normal`|
+|`serverpodStartTimeout`|`Duration?`|`Duration(seconds: 30)`|
+|`testGroupTagsOverride`|`List<String>?`|`['integration']`|
+
+### `applyMigrations`
+
+Wether pending migrations should be applied when starting Serverpod. Defaults to `true`.
```

Comment:
Suggestion: It is `Whether` right?

My google says: The short answer is that wether is a male goat or sheep, whether means “if,” 

```suggestion
Whether pending migrations should be applied when starting Serverpod. Defaults to `true`.
```